llvm/lib/CodeGen/MLRegAllocEvictAdvisor.cpp

```diff
@@ -803,6 +805,21 @@ MCRegister MLEvictAdvisor::tryFindEvictionCandidate(
   }
   assert(CandidatePos < ValidPosLimit);
   (void)ValidPosLimit;
+
+  // Update information about how many times the virtual registers being
+  // evicted have been evicted.
+  if (CandidatePos == CandidateVirtRegPos) {
+    VirtRegEvictionCounts[VirtReg.reg()] += 1;
```

Comment:
wrap this increment in a small private function, like `incrementEvictCount`, for readability. The function wouldn't need to be const. Is there a reason to keep the field `mutable`?

Reply:
`tryFindEvictionCandidate` is marked `const`, so a function like `incrementEvictCount` would also need to be `const`, unless we refactor those functions to be non-const. That's also why the field is mutable.

Reply:
Wrapped it in a function.